content/en/user-guide/aws/lambda/index.md

```diff
@@ -3,84 +3,110 @@ title: "Lambda"
 linkTitle: "Lambda"
 categories: ["LocalStack Community", "LocalStack Pro"]
 description: >
-  Get started with local development and testing of AWS Lambdas on LocalStack
+  Get started with AWS Lambdas on LocalStack
+tags:
+- lambda
+- aws-lambda
+- lambda-hot-reloading
+- lambda-debugging
+- localstack-lambda
+- serverless
+- lambda-layers
 aliases:
   - /aws/lambda/
+  - /references/lambda-executors/
+  - /references/lambda-provider-v2/
```

Comment:
FYI: The page `lambda-provider-v2` had two more (deprecated aliases):
* /references/lambda-asf-provider/
* /references/lambda-v2-provider/